eng/config/rulesets/Shipping.ruleset

```diff
@@ -55,8 +56,11 @@
     <Rule Id="CA1823" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/20404: Enable rule CA1823 (remove unused field) once we have a fixer for it -->
     <Rule Id="CA1824" Action="None" /> <!-- mark assemblies with NeutralResourcesLanguageAttribute -->
     <Rule Id="CA1827" Action="None" /> <!-- https://github.com/dotnet/roslyn/issues/37959: DoNotUseCountWhenAnyCanBeUsed -->
+    <Rule Id="CA1834" Action="None" /> <!-- Consider using 'StringBuilder.Append(char)' when applicable -->
```

Comment:
Do we want to open tracking issues to enable rules which are useful, but just not in scope of this PR?

Reply:
I can for this one. Are there any others?

Reply:
CA1835, CA2249?

Reply:
Issues are filed